libnmstate/netapplier.py

```diff
@@ -163,8 +162,7 @@ def _apply_ifaces_state(
         # finish before proceeding with other actions.
         # TODO: https://nmstate.atlassian.net/browse/NMSTATE-103
         time.sleep(5)
-        if not nmclient.can_disable_ipv6():
-            _disable_ipv6(current_state)
+        _disable_ipv6(current_state)
```

Comment:
Please also remove the use of `_disable_ipv6()` which is not required on NM 1.20+

Reply:
Oops. You are right, done! Thanks :-)